Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service.ts

```diff
@@ -230,7 +240,26 @@ export class AzureAppService {
 
         return isNewValueUpdated;
     }
+
+    public async patchApplicationSettingsSlot(addProperties: any): Promise<any> {
+        var appSettingsSlotSettings = await this.getSlotConfigurationNames();
+        var isNewValueUpdated: boolean = false;
+        for(var key in addProperties) {
+            if(addProperties[key].slotSetting == true) {
+                if((appSettingsSlotSettings.properties.appSettingNames.length == 0) || (!appSettingsSlotSettings.properties.appSettingNames.includes(addProperties[key].name))) {
```

Comment:
nit let appSettingNames= connectionStringSlotSettings.properties.appSettingNames;
Also is it possible that connectionSTringNames is null or properties is null

Reply:
Included the check for null already